benchmark/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM nvidia/cuda:12.2.0-runtime-ubuntu22.04
+
+RUN ln -snf /usr/share/zoneinfo/Europe/Zurich /etc/localtime && \
+    echo Europe/Zurich > /etc/timezone && \
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    apt-get update && \
+    apt-get install -y \
+        python3 python3-pip \
+        && \
+    apt-get autoremove && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY ./requirements.txt giotto-deep/
+
+RUN cd giotto-deep && \
```

Comment:
Better set WORKDIR

Reply:
Yes, but if I remember well, I had troubles with WORKDIR in this situation. So "cd" it is :)